src/components/Header/Search.js

```diff
@@ -1,11 +1,27 @@
 import React from 'react';
+import { Formik, Form, Field } from 'formik';
+import { useHistory } from 'react-router-dom';
 import '../../styles/search.scss';
 
-const Search = () => (
-    <form className="search">
-        <input type="text" className="search__input" placeholder="What do you want to watch?"/>
-        <button type="submit" className="search__submit">Search</button>
-    </form>
-);
+const Search = () => {
+    const browserHistory = useHistory();
+
+    return <Formik
+        initialValues={{searchMovie: ''}}
+        onSubmit={(values) => {
+            const query = values.searchMovie;
+            if (query) {
+                browserHistory.push({
+                    pathname: `/search/${values.searchMovie}`,
+                })
+            }
+        }}
+    >
```

Comment:
can this function be moved from return statement and be declared above return statement?

Reply:
Fixed, thank you.